x/foundation/keeper/keys.go

```diff
@@ -13,15 +13,15 @@ var (
 	paramsKey         = []byte{0x00}
 	foundationInfoKey = []byte{0x01}
 
-	// treasuryKey = []byte{0x10}
+	memberKeyPrefix          = []byte{0x10}
+	previousProposalIDKey    = []byte{0x11}
+	proposalKeyPrefix        = []byte{0x12}
+	proposalByVPEndKeyPrefix = []byte{0x13}
+	voteKeyPrefix            = []byte{0x14}
 
-	validatorAuthKeyPrefix = []byte{0x20}
+	grantKeyPrefix = []byte{0x20}
 
-	memberKeyPrefix          = []byte{0x30}
-	previousProposalIDKey    = []byte{0x31}
-	proposalKeyPrefix        = []byte{0x32}
-	proposalByVPEndKeyPrefix = []byte{0x33}
-	voteKeyPrefix            = []byte{0x34}
+	// treasuryKey = []byte{0x??}
```

Comment:
Why this property is commented?

Reply:
The treasury query of x/foundation does not support pagination yet. To support it, I may use treasury key and track the module's balance.